src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  padding: 30px $padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: (7 * $cell-width) + (6 * $gap) + (2 * $padding);
+
+  &--start-day {
+    @each $day, $count in $start-days {
+      &--#{$day} :first-child {
+        margin-left: ($count * $cell-width) + ($count * $gap);
+      }
+    }
+  }
+
+  &--month-length {
+    @for $length from $min-month through $max-month {
+      &--#{$length} :nth-child(n + #{$length + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    width: $cell-width;
+    height: $cell-height;
+    border: 1px solid #000;
+
+    background: #eee;
+    text-align: center;
+    line-height: $cell-height;
+    font-size: 30px;
+
+    transition: transform 0.5s;
+
+    &:hover {
+      background: pink;
+      cursor: pointer;
```

Comment:
this style should be added to `.calendar__day` without `hover` state

Reply:
Sorry, can't understand what you mean. I read through the task and found the following: 
  Each day is a grey (#eee) 100px square (including 1px black border)
  On hovering over a cell:
    - cursor should become pointer
    - The hovered cell has to become pink
  
 Could you explain what should I do in more details?

Reply:
Sorry, I can't understand what you mean. I read through the task and found the following: 
  Each day is a grey (#eee) 100px square (including 1px black border)
  On hovering over a cell:
    - cursor should become pointer
    - The hovered cell has to become pink
  
 Could you explain what should I do in more details?